patroni/dcs/consul.py

```diff
@@ -191,16 +191,18 @@ def touch_member(self, data, **kwargs):
             return True
 
         try:
-            self._client.kv.put(self.member_path, data, acquire=self._session)
+            args = {'cas': None} if kwargs.get('permanent') else {}
```

Comment:
Method signature has to be:
```python
    def touch_member(self, data, permanent=False, **kwargs):
```

`cas`? It's for Compare-And-Save operations...
If you want to create permanent member, you should NOT specify `acquire`:
```python
    args = {} if permanent else {'acquire': self._session}
    self._client.kv.put(self.member_path, data, **args)
```